### PR TITLE
Fix for false positive on paragraph elements font-family

### LINKED ISSUE #6286: Set the Font Family of an Element passes with incorrect code
A camper pointed out that this challenge can be passed with the following code:

```
var code = "formatted";
<style>
  .red-text {
    color: red;
    font-family: Monospace;
  }

  p {
    font-size: 16px;
  }
</style>

<h2 class="red-text">CatPhotoApp</h2>

<p class="red-text">Kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff.</p>
<p>Purr jump eat the grass rip the couch scratched sunbathe, shed everywhere rip the couch sleep in the sink fluffy fur catnip scratched.</p>
```

This does not meet the objectives of the lesson (*make all of your p elements use the Monospace font*), but both Takumar and I were able to recreate the issue.

Suggested solution: adjusting the test
```
"tests": [
        "assert($(\"p\").css(\"font-family\").match(/monospace/i), 'Your <code>p</code> elements should use the font <code>Monospace</code>.')"
      ]
```

When discussing with Takumar, he found that the next challenge, "Import a Google Font", has a similar test and could also be adjusted. (I did not verify his finding, as i don't know how to check the tests, but trust his assessment :) )

### PR DESCRIPTION
**Pull request by ghost**
started new branch as last one was a mess.
this fix removes the false positive assert test mentioned in bug below
closes #6286

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/html5-and-css.json [hunk 1]
```diff
@@ -540,7 +540,7 @@
         "<p>Purr jump eat the grass rip the couch scratched sunbathe, shed everywhere rip the couch sleep in the sink fluffy fur catnip scratched.</p>"
       ],
       "tests": [
-        "assert($(\"p\").css(\"font-family\").match(/monospace/i), 'message: Your <code>p</code> elements should use the font <code>Monospace</code>.');"
+        "assert($(\"p\").not(\'.red-text\").css(\"font-family\").match(/monospace/i), 'message: Your <code>p</code> elements should use the font <code>Monospace</code>.');"
```
Comment (SaintPeter) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: Looks like you have a mismatched quote here.  Should be `not(\".red-text\").`

**Comment by SaintPeter** [comment]
Your branch name is also not in the correct format.  Should be something like `fix/font-family-tests`.

Be sure to `git commit --amend` and then `git push -f` to update your branch.

More details here:
https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md

**Comment by robjloranger** [comment]
I tried to change the name too but then I was creating new branches on pushing changes.

**Comment by SaintPeter** [comment]
Looking good!

**Pull request merged**